plugins/basic/nexus-ui-extjs3-plugin/src/main/resources/static/js/Sonatype/utils.js

```diff
@@ -818,7 +817,6 @@ define('Sonatype/utils',['../extjs', 'Nexus/config', 'Nexus/util/Format', 'Sonat
                   method : 'GET',
                   url : Sonatype.config.repos.urls.logout,
                   callback : function(options, success, response) {
-                    ns.clearCookie('JSESSIONID');
```

Comment:
FTR I've no idea why the UI does/did this... but we should be sure to test that removing this (and other related thing) doesn't break anything.